libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -1307,19 +1307,26 @@ AP_GPS_UBLOX::broadcast_configuration_failure_reason(void) const {
 /*
   return velocity lag in seconds
  */
-float AP_GPS_UBLOX::get_lag(void) const
+bool AP_GPS_UBLOX::get_lag(float &lag_sec) const
 {
     switch (_hardware_generation) {
+    case UBLOX_UNKNOWN_HARDWARE_GENERATION:
+        lag_sec = 0.22f;
+        // always bail out in this case, it's used to indicate we have yet to recieve a valid
+        // hardware generation
+        return false;
```

Comment:
I'm not quite sure this is the correct approach. How does this work with auto config turned off?